module/Activity/src/Activity/Service/Activity.php

```diff
@@ -306,6 +131,43 @@ public function createActivity(array $params, $dutch, $english)
     }
 
     /**
+     * Find the organ the activity belongs to, and see if the user has permission to create an activity
+     * for this organ.
+     *
+     * @param \User\model\User $user The user creating the activity
+     * @param int $organId The id of the organ associated with the activity
+     * @return Organ The organ associated with the activity, if the user is a member of that organ
+     * @throws \User\Permissions\NotAllowedException if the user is not a member of the organ specified
+     */
+    protected function findOrgan(\User\model\User $user, $organId)
+    {
```

Comment:
I believe this whole function could be replaced by something like:

```
$organ = null;
try {
    $organ = $organService->getOrgan($id);
} catch (\Exception $e) {
    // Organ not found
}

if (!$decisionService->canEditOrgan($organ)) {
    // User is not allowed to create activity for this organ
}

return $organ;
```

It would make it slightly better understandable in my opinion.